# For uwb_localization
# In case of any infinate value while calculating that causes nan error,
# the min and max ranges need to be set.
following_node:
    ros__parameters:
        tag_id: 1

        # minimum and maximum uwb raw range in meters
        min_raw_range: 0.3
        max_raw_range: 3.6 # 2 times of lengh of MIH

        # position if anchor [x, y, z] in meters
        # Anchor1 and anchor2 need to be on the same horizontal level (z)
        # Anchor3 and anchor4 need to be on the same vertiacal axis (y
        anchor_1:
          x: 0.80
          y: 0.30
          z: 0.35
        anchor_2:
          x: 0.80
          y: -0.30
          z: 0.35
        anchor_3:
          x: 0.90
          y: 0.0
          z: 0.96
        anchor_4: 
          x: 0.90
          y: 0.0
          z: 0.52
